tests/dict_import_export_tests.py

```diff
@@ -87,11 +92,20 @@ def create_table(self, name, schema="", id=0, cols_names=[], metric_names=[]):
         return table, dict_rep
 
     def create_druid_datasource(self, name, id=0, cols_names=[], metric_names=[]):
-        name = "{0}{1}".format(NAME_PREFIX, name)
         cluster_name = "druid_test"
+        cluster = (
```

Comment:
This is repeated below. Can we create a utility method instead.

Reply:
Found one that existed 🎉